packages/cra-template-imodeljs/template/src/logo.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```

Comment:
havent pulled locally, so dont know what this actually looks like, but want to confirm this svg along with the logo*.png and favicon should be an imodeljs logo